addons/mrp/models/mrp_bom.py

```diff
@@ -476,6 +483,14 @@ def _update_order_line_info(self, product_id, quantity, child_field=False, **kwa
 
         return self.env['product.product'].browse(product_id).standard_price
 
+    def _get_action_add_from_catalog_extra_context(self):
+        return {
+            **super()._get_action_add_from_catalog_extra_context(),
+            'display_uom': self.env.user.has_group('uom.group_uom'),
```

Comment:
could be in mixin no?

Reply:
Hello @amoyaux ,

Yes, that's right. Then we should also apply the same approach for the sale_order and purchase_order model ?, as they also have similar methods.

https://github.com/odoo/odoo/blob/master/addons/purchase/models/purchase_order.py#L847
https://github.com/odoo/odoo/blob/master/addons/sale/models/sale_order.py#L1182

Thanks.